crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -561,7 +561,8 @@ impl AuthorityPerEpochStore {
         );
 
         let zklogin_env = match chain_identifier.chain() {
-            Chain::Mainnet => ZkLoginEnv::Prod,
+            // Testnet and mainnet are treated the same since it is permanent.
+            Chain::Mainnet | Chain::Testnet => ZkLoginEnv::Prod,
```

Comment:
are you sure this won't fork testnet when it is released? i forget exactly what the env param is for

Reply:
mainnet | testnet -> prod vk
devnet -> test vk (migrated to a new fake vk)
see: https://github.com/MystenLabs/fastcrypto/commit/2dc1d6e23cef72880a0fcb699c192b49a240d7e5#diff-4dc4e0f52881c37bef9ab8dd9e0ca92837bc97a5658dd84e0f344467ed72e260R265)